### PR TITLE
docs: Install Tembo Self Hosted on Azure

### PR DESCRIPTION
**Pull request by vrmiguel**

### CODE REVIEW

**Comment by ryw** [comment]
any reason not to merge this @ianstanton @vrmiguel ?

File: src/content/docs/product/software/tembo-self-hosted/install-on-azure.mdx [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+export CLUSTER_NAME=your-cluster-name
+
+az aks create --resource-group $RESOURCE_GROUP --name $CLUSTER_NAME --node-count 3 --node-vm-size Standard_D4s_v3 --enable-managed-identity --location eastus
+```
+
+Feel free to adjust the parameters as needed to suit your requirements.
+
+For further information, see [Quickstart: Deploy an Azure Kubernetes Service (AKS) cluster](https://learn.microsoft.com/en-us/azure/aks/learn/quick-kubernetes-deploy-cli).
+
+## Step 4: Install the Azure Disk CSI driver
```
Comment (ianstanton) [inline src/content/docs/product/software/tembo-self-hosted/install-on-azure.mdx:47]: When I ran through an installation, I did not need to perform these steps. We should circle back and confirm this is not necessary.

File: src/content/docs/product/software/tembo-self-hosted/install-on-azure.mdx [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+kubectl apply -f traefik-postgres-catch-all.yaml
+```
+
+We now need to configure Traefik to handle Postgres connections appropriately. See the [example values](TODO) for reference.
+
+Finally, install Traefik:
+
+```shell
+helm install traefik traefik/traefik --namespace traefik --version 20.8.0 -f traefik-values.yaml
+```
```
Comment (ianstanton) [inline src/content/docs/product/software/tembo-self-hosted/install-on-azure.mdx:131]: These two charts will be added as optional sub-charts in the self-hosted helm chart soon. We'll come back and update once that is done.

File: src/content/docs/product/software/tembo-self-hosted/install-on-azure.mdx [hunk 3]
```diff
@@ -0,0 +1,292 @@
...
+                  More info: https://doc.traefik.io/traefik/v2.11/middlewares/tcp/ipallowlist/
+                properties:
+                  sourceRange:
+                    description: SourceRange defines the allowed IPs (or ranges of
+                      allowed IPs by using CIDR notation).
+                    items:
+                      type: string
+                    type: array
+                type: object
+```
```
Comment (ianstanton) [inline src/content/docs/product/software/tembo-self-hosted/install-on-azure.mdx:205]: This will also be dropped once we add `traefik` as a sub-chart.

**Review by ianstanton — APPROVED** [review]
This is in a good enough state to merge. We can circle back with updates once they are ready.

**Pull request merged**